gatsby-config.js

```diff
@@ -53,6 +53,31 @@ module.exports = {
     // To learn more, visit: https://gatsby.dev/offline
     // Docs: https://www.gatsbyjs.org/packages/gatsby-plugin-offline/
     // `gatsby-plugin-offline`,
+    {
+      // Docs: https://www.gatsbyjs.com/plugins/gatsby-plugin-react-i18next/
+      resolve: `gatsby-plugin-react-i18next`,
+      options: {
+        path: `${__dirname}/locale`,
+        languages: [`en`, `cs`],
+        defaultLanguage: `cs`,
+        redirect: false,
+        i18nextOptions: {
+          defaultNS: 'translation',
+        },
+      },
+    },
+    {
+      // Docs: https://www.npmjs.com/package/gatsby-plugin-translate-urls
+      resolve: `gatsby-plugin-translate-urls`,
```

Comment:
Díky za nalezení řešení. Ověřil jsem případ vnořených url (např. `/o-nas/komunita`) a generování pro projekty. Vše funguje a vypadá to použitelně. Jediná škoda je nutnost mít JSON pro oba jazyky a ne třeba jen pro ANJ. Na druhou stranu zase můžeme všechno v `pages` mít v ANJ. 

Navrhuji ještě jednu změnu: Vyhodit definici cest a jejich překladů z AirTable a JSON natvrdo vložit do repozitáře. Jedná se o věc, která je navázaná na složku `pages` a `gatsby-node.js` a myslím, že je vhodné mít definici URLs více pod kontrolou a zamezit úpravě bez znalosti důsledků (změna URL je breaking change). Co myslíš? 

V každém případě super počin. 👍

Reply:
Souhlas, dava smysl, hodim to jako separe definice a vyhodim pryc z Airtablu.